packages/strapi-plugin-content-type-builder/admin/src/containers/App/actions.js

```diff
@@ -0,0 +1,40 @@
+/*
+ *
+ * App actions
+ *
+ */
+import { size, forEach } from 'lodash';
+import { MODELS_FETCH, MODELS_FETCH_SUCCEEDED } from './constants';
+
+export function modelsFetch() {
+  return {
+    type: MODELS_FETCH,
+  };
+}
+
+export function modelsFetchSucceeded(data) {
+  const modelNumber = size(data.models) > 1 ? 'plural' : 'singular';
+  const sections = [];
```

Comment:
Same here, this is a common pattern. You are populating a variable, so it will be better to use map instead of foreach.

```js
const sections = data.models
   .map(model => { icon: 'fa-caret-square-o-right', name: model.name })
   .push({ icon: 'fa-plus', name: 'button.contentType.add' })
```